packages/product-download-page/partials/download-cards/style.module.css

```diff
@@ -114,6 +114,8 @@
     padding: 0;
     background-color: inherit;
     white-space: normal;
+    font-size: 0.844rem;
+    font-family: var(--font-monospace);
```

Comment:
Backfilling `code` styles. I find the setup here a little odd with `<pre>` but no `<code>`, but works alright so just trying to touch it as little as possible to keep the scope of this PR sane.